pkg/operator/sync.go

```diff
@@ -192,6 +195,35 @@ func (c *csiDriverOperator) syncRBAC(instance *v1alpha1.EBSCSIDriver) error {
 	return nil
 }
 
+func (c *csiDriverOperator) syncCredentialsRequest(instance *v1alpha1.EBSCSIDriver) (*unstructured.Unstructured, error) {
```

Comment:
Not a blocker as such - but is it a problem that we create the CredentialRequest but for some reason the secret did not get created and now all our deployments can't start. User will see a cryptic error which won't make sense to them.

Reply:
Added wait for Secret to appear & sending events for it.